packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -379,7 +388,8 @@ export class ExternalRequest {
       ) {
         continue
       }
-      let linked = basicProcessing(row, linkedTable)
+
+      let linked = basicProcessing(row, linkedTable, true)
```

Comment:
Can we change this to an object instead? You can't see what the parameter name is.

I.e. 
```js
{
  row,
  linkedTable,
  isLinked: true
}
```

Reply:
Sorry - didn't see this comment, I made a similar comment further up.

Reply:
Yup, I totally agree with this kind of best practices. I just did want to touch as little things as possible on the code, as I don't want to make a bug hard to PR. But happy to change it :)